eth_account/account.py

```diff
@@ -222,13 +227,12 @@ def privateKeyToAccount(self, private_key):
         return LocalAccount(key, self)
 
     @combomethod
-    def recoverHash(self, message_hash, vrs=None, signature=None):
+    def recover_message(self, signable_message: SignableMessage, vrs=None, signature=None):
```

Comment:
What's the reason for using `recover_message` as the function name? I don't have the most complete knowledge of all the cryptography terminology, but something like `recover_address` is more intuitive to someone like me

Reply:
Right, so this is "running `recover` on a message", not "recover the message". I can definitely see how if you're not swimming in `ecrecover` all day it might not be obvious. Hopefully the docs explain what it does enough to avoid confusion. Whether you are running recover on the transaction or on the message, you are always recovering an address, so `recover_address` is ambiguous. `recover_on_message` feels pretty unwieldy to me. I don't really have any better ideas.